go/base/management/commands/go_manage_application.py

```diff
@@ -1,21 +1,15 @@
 import uuid
 from optparse import make_option
 
-from django.core.management.base import BaseCommand, CommandError
+from go.base.command_utils import BaseGoAccountCommand, CommandError
 
-from go.base.utils import vumi_api_for_user
-from go.base.command_utils import get_user_by_email
 
-
-class Command(BaseCommand):
+class Command(BaseGoAccountCommand):
     help = "Give a Vumi Go user access to a certain application"
 
     LOCAL_OPTIONS = [
-        make_option('--email-address',
-            dest='email-address',
-            help='Email address for the Vumi Go user'),
-        make_option('--application-module',
-            dest='application-module',
+        make_option('--application_module',
```

Comment:
The option we accept from the command line should be `--application-module`, not `--application_module`. We only want to change the internal dict key, not the thing we have to type to get the behaviour we want.